config_0:
  stick_left_H:         STEERING_X
  stick_left_V:         STEERING_Z
  stick_right_H:        none
  stick_right_V:        none
  button_up_down:       none
  button_left_right:    none
  button_A:             none
  button_B:             none
  button_Y:             none
  button_X:             none
  button_l1:            none
  button_r1:            none
  button_l2:            none
  button_r2:            none
  button_l3:            none
  button_r3:            none
  button_option:        STATE
  button_start:         none
  button_home:          none
config_1:
  stick_left_H:         MFRAME
  stick_left_V:         BOOM
  stick_right_H:        BLOCK
  stick_right_V:        ARM
  button_up_down:       none
  button_left_right:    none
  button_A:             TOHKU_PITCH_UP
  button_B:             TOHKU_ROLL_UP
  button_Y:             TOHKU_PITCH_DOWN
  button_X:             TOHKU_ROLL_DOWN
  button_l1:            none
  button_r1:            none
  button_l2:            none
  button_r2:            none
  button_l3:            none
  button_r3:            none
  button_option:        STATE
  button_start:         none
  button_home:          none
config_2:
  stick_left:           /RTRDoubleArmV7/base_controller/cmd_vel     # 左スティック
  stick_right:          none                                        # 右スティック
  button_up_down:       none                                        # 十字キー 上下
  button_left_right:    none                                        # 十字キー 左右
  button_A:             none                                        # ×
  button_B:             none                                        # ❍
  button_X:             none                                        # □
  button_Y:             none                                        # △
  button_l1:            none                                        # L1
  button_r1:            none                                        # R1
  button_l2:            none                                        # L2
  button_r2:            none                                        # R2
  button_l3:            none                                        # L3
  button_r3:            none                                        # R3
  button_option:        STATE                                       # option
  button_start:         none                                        # start
  button_home:          none                                        # home